tests/data/arborator/16_metadata/metadata.included.tsv

```diff
@@ -1,7 +1,7 @@
 Sample	Outbreak Code	Organism	Subtype	Country of Collection	Serovar	Patient Age (years)	Date	Source Type	Special	metadata_9	metadata_10	metadata_11	metadata_12	metadata_13	metadata_14	metadata_15	metadata_16	gas_denovo_cluster_address
-S1	1	Escherichia coli	EHEC/STEC	Canada	O157:H7	21	2024/05/30	beef	True	s1-meta9	s1-meta10	s1-meta11	s1-meta12	s1-meta13	s1-meta14	s1-meta15	s1-meta16	1|1.1.1.1.1.1.1.1.1.1
```

Comment:
This is probably related to the CSVs being read as strings, not letting pandas interpret them as Boolean objects, so they're not the bool.tostr capitalized.